sqlsorcery.py

```diff
@@ -0,0 +1,100 @@
+from os import getenv
+from google.oauth2 import service_account
+import pandas as pd
+import pandas_gbq
+from sqlalchemy import create_engine, Table, MetaData
+from sqlalchemy.engine.url import URL
+from sqlalchemy.exc import NoSuchTableError
+from sqlalchemy.schema import DropTable
+
+
+class Connection:
```

Comment:
Do we have the performance optimization options for postgres/mssql here at all?

Reply:
I removed them as this fork is designed to be used with BigQuery specifically, but I have no problem adding them back in.